packages/rocketchat-livechat/server/hooks/sendToCRM.js

```diff
@@ -102,3 +102,13 @@ RocketChat.callbacks.add('livechat.leadCapture', (room) => {
 	}
 	return sendToCRM('LeadCapture', room, false);
 }, RocketChat.callbacks.priority.MEDIUM, 'livechat-send-crm-lead-capture');
+
+RocketChat.callbacks.add('livechat.visitorTransfer', function(room, transferData) {
+	if (!RocketChat.settings.get('Livechat_webhook_on_visitor_transfer')) {
+		return room;
+	}
+	if (transferData.receivedByAPI) {
```

Comment:
It's related to this: https://github.com/RocketChat/Rocket.Chat/pull/11456#discussion_r203741460